ui/packages/consul-peerings/app/components/consul/peer/form/token/actions/index.hbs

```diff
@@ -0,0 +1,18 @@
+<div
+  class={{class-map
+    'consul-peer-form-token-actions'
+  }}
+  ...attributes
+>
+  <Action
+    ...attributes
+    {{with-copyable @token}}
+  >
+    Copy token
+  </Action>
+  <Action
+    {{on 'click' @onclose}}
```

Comment:
I'm not seeing the @onclose action being passed when yielding this component. do we want to handle the case for when users don't provide @onclose?

Reply:
👍 I can add add an `optional`, 2 tics

Reply:
Hmm honestly, I would rather this errored if you don't add an `@onclose` otherwise the button doesn't do anything. This is also a single usecase component, mainly a component for better organisation, so highly unlikely that this will never be set, and again if that is the case then I would rather it wasn't silent. Are you ok me leaving this as is?